todo-plainJS.js

```diff
@@ -0,0 +1,154 @@
+let taskList = JSON.parse(localStorage.getItem('taskList')) ?? [];
+
+window.addEventListener('DOMContentLoaded', () => {
+    drawHTML(filterTasks());
+    refreshCount();
+});
+
+form.addEventListener('submit', (event) => {
+    event.preventDefault();
+    const content = task_content.value;
+
+    if (content === '') {
+        console.log('Input is empty');
+    } else {
+        createTask(content);
+        createTaskHTML(content);
+    }
+
+    saveTaskList();
+    refreshCount();
+    task_content.value = '';
+});
+
+for (let item of filters.children) {
+    item.addEventListener('click', () => {
+        switch (item.id) {
+            case 'filter_done':
+                drawHTML(filterTasks('done'));
+                refreshCount(filterTasks('done'));
+                break;
+            case 'filter_undone':
+                drawHTML(filterTasks('undone'));
+                refreshCount();
+                break;
+            default:
+                drawHTML(filterTasks());
+                refreshCount();
+                break;
+        }
+    });
+}
+
+todo.addEventListener('click', (event) => {
```

Comment:
I do not like this approach, it would be better to assign eventListener to the task, not to the parent container.